REQUIRE

```diff
@@ -1,6 +1,6 @@
 julia 0.4
 Nettle 0.2
 JSON 0.5
-ZMQ 0.3.1
+ZMQ 0.3.4
```

Comment:
is this fixing a deprecation warning or an error? if it's just fixing a deprecation warning, it's too tight